### PR TITLE
changes for implementing nemo in cesm2.3

### PR DESCRIPTION
**Pull request by daniele-peano**
[ Description of the changes in this Pull Request. It should be enough
information for someone not following this development to understand.
Lines should be wrapped at about 72 characters. Please also update
the CIME documentation, if necessary, in doc/source/rst and indicate
below if you need to have the gh-pages html regenerated.]

Test suite:
Test baseline:
Test namelist changes:
Test status: [bit for bit, roundoff, climate changing]

Fixes [CIME Github issue #]

User interface changes?:

Update gh-pages html (Y/N)?:

### CODE REVIEW

File: scripts/Tools/mkDepends [hunk 1]
```diff
@@ -346,7 +346,7 @@ foreach $f (keys %file_includes) {
     }
 
     # Expand
-    for ($i = 0; $i <= $#expand_incs; ++$i) {
```
Comment (jedwards4b) [inline scripts/Tools/mkDepends:349]: why is this change required?

**Review by jedwards4b — APPROVED** [review]

File: scripts/Tools/mkDepends (hunk 1 above)
Comment (daniele-peano) [reply scripts/Tools/mkDepends:349]: When running the model on Zeus with NEMO, the build of NEMO becomes slow and uses a lot of memory with the original code. 
With the suggested code this problem is solved.

**Review by jedwards4b — APPROVED** [review]

**Comment by jedwards4b** [comment]
Minor changes needed for the NEMO ocean model.
